doc/md/getting-started/manual-installation.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Manual Installation
+slug: /manual-installation
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+### Manual Installation
+
+If you don't trust the shell installation script or want to do a manual installation for some other reason, follow the
+below instructions.
```

Comment:
The sentence feels odd to me. Also, I feel like we can add it under CLI reference and not in the getting started, but add a link to it in the `InstalltionInstruction` component. 
![image](https://user-images.githubusercontent.com/7413593/208671236-9f92fbbb-0760-4778-ace5-64adde0b97f6.png)

Reply:
I don't think so. I think everywhere we have the installation instructions it should be clear that the user can install without the script as well. 

I'd be okay with dropping the sentence though.

Reply:
Why not add it to the `InstalltionInstruction` as a "manual installation" tab? Since this is quite a rare use case I don't think it should reside in the second document.